dev/Repeater/APITests/FlowLayoutTests.cs

```diff
@@ -146,6 +146,15 @@ public void ValidateGridLayout()
                         ((UniformGridLayout)panel.Layout).MinColumnSpacing = minColumnSpacing;
                         Content.UpdateLayout();
                         ValidateGridLayoutChildrenLayoutBounds(om, (i) => panel.Children[i], itemMinorSize, itemMajorSize, minRowSpacing, minColumnSpacing, panel.Children.Count, panel.DesiredSize);
+
+                        minRowSpacing = 5;
```

Comment:
It'd be helpful to have a comment (or log statement) that describes what this block is trying to validate.